articles/introduction-to-editing.mdx

```diff
@@ -0,0 +1,138 @@
+_uuid: d8cc7ee0-cec3-473e-a85d-0fcef4aaf76d
+_created_at: 2023-11-14T13:06:25+13:00
+title: Introduction to Editing
+nav_title: Introduction
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: >-
+  Learn how editing works in CloudCannon, including configuration, collections,
+  editing and saving changes, and content collaboration.
+tags: []
+related_articles: []
+related_links: []
+explicit_canonical:
+---
+Editing on CloudCannon enables anyone on your team to contribute to your Git repository, regardless of technical expertise.
+
+Traditional Content Management Systems (CMSs) use databases to store your content. CloudCannon takes a different approach. By storing your content in files alongside your code, your content can benefit from the Git workflow (versioning, accountability, branching, etc.).
+
+With CloudCannon, developers can configure their local environment just the way they want it, and non-developers can edit in the app with ease. You have control over how you configure your experience.
+
+<comp.Notice info_type="info">
+  If this is your first time creating a site on CloudCannon, or you need a refresher, check out our [Starter Guides](/documentation/guides/) — we have one for every SSG we support!
+</comp.Notice>
+
+In the Editing section of our documentation, we cover how to:
+
+* [Create a global configuration file](/documentation/articles/create-a-global-configuration-file/) — create the file at the heart of CloudCannon customization.
+* [Define your collections](/documentation/articles/define-your-collections/) — groups of related files that form the start of any new site.
+* Editing interfaces — the [Visual](/documentation/articles/introducing-the-visual-editor/), [Content](/documentation/articles/introducing-the-content-editor/), [Data](/documentation/articles/introducing-the-data-editor/), and [Source](/documentation/articles/introducing-the-source-editor/) Editors.
+* [Edit your files](/documentation/articles/edit-your-files/) — cooperate on file changes with live editing.
+* [Save your changes](/documentation/articles/save-your-changes/) — learn how to review changes made by your team and save back to your git repository.
+* [Configure your inputs](/documentation/articles/configure-your-inputs/) — configure your structured data to create an intuitive interface.
+* [Create a structure](/documentation/articles/create-a-structure/) — create predefined objects to add to your content.
+* Create a Snippet — create structured components in Markup using [Docusaurus](/documentation/articles/snippets-using-docusaurus-components/), [Eleventy](/documentation/articles/snippets-using-eleventy-shortcodes/), [Hugo Shortcodes](/documentation/articles/snippets-using-hugo-shortcodes/), [MDX](/documentation/articles/snippets-using-mdx-components/), or [Python Markdown](/documentation/articles/snippets-using-python-markdown/).
```

Comment:
We should either name these after the SSG (`Eleventy`) or concept within the SSG (`Hugo Shortcodes`)

```suggestion
* Create a Snippet — create structured components in Markup using:
  * [Docusaurus Components](/documentation/articles/snippets-using-docusaurus-components/)
  * [Eleventy Shortcodes](/documentation/articles/snippets-using-eleventy-shortcodes/)
  * [Hugo Shortcodes](/documentation/articles/snippets-using-hugo-shortcodes/)
  * [MDX Components](/documentation/articles/snippets-using-mdx-components/)
  * [Python Markdown](/documentation/articles/snippets-using-python-markdown/).
```